src/normal.c

```diff
@@ -4074,6 +4074,9 @@ nv_gotofile(cmdarg_T *cap)
 	return;
 #endif
 
+    if (!check_can_set_curbuf_disabled())
+      return;
+
```

Comment:
Used in these tests

Test_normal_gF
Test_normal_gf
Test_normal_square_bracket_left_f
Test_normal_square_bracket_right_f